test/kwokctl/logs.yaml

```diff
@@ -0,0 +1,26 @@
+kind: Logs
+apiVersion: kwok.x-k8s.io/v1alpha1
+metadata:
+  name: fake-pod
+  namespace: other
+spec:
+  logs:
+    - containers:
+      - fake-pod
+      logsFile: /tmp/kwok/kwok.log
```

Comment:
If this is a relative path, then it is either relative to the configuration file itself or relative to pwd?

Reply:
I added it to the `config.Load`
It seemed like a nice place since the defaults were applied there, and at that point, we know what is the working directory.
Now the linter is complaining about it, but to me, it seemed like the most appropriate place :smile:

Reply:
But I think we are still going to bump into the same linter issue when we extend another config option like `Attach`

Reply:
Yes, I think `attach` is actually `logs -f` if the input is ignored.